core-services/prow/02_config/openshift/cli-manager/_prowconfig.yaml

```diff
@@ -1,5 +1,21 @@
 tide:
   queries:
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/cli-manager
```

Comment:
This lines are duplicates. Should not this go under `core-services/prow/02_config/openshift/cli-manager-operator/_prowconfig.yaml` instead as `repos: ["openshift/cli-manager-operator"]`?

Reply:
It is interesting because I didn't want to make any changes on this file but `make update` always modified this. I'll check that what I'm doing wrong.

Reply:
Manually removed duplicate and successfully ran `make update`.

Reply:
and thanks for review :)